constant.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "constant",
+    "summary": "Define a constant value",
+    "description": "Define a constant value that can be reused in multiple places of the process graph",
+    "categories": [
+        "math > constants"
+    ],
+    "parameters": [
+        {
+            "name": "x",
+            "description": "The value of the constant.",
+            "schema": {
+                "type": "number"
```

Comment:
I think it makes sense to allow any data type here, right? Defining a larger array or string once makes sense, too.

Reply:
I couldn't come up with a good use case for strings, but arrays make sense indeed

Reply:
Maybe something like a re-occurring band or dimension name? Not sure, but also don't see a reason to restrict this to any data type (although boolean and null seems really weird).

Reply:
I think the schema should simply be:

```json

            "schema": {
                "description": "Any data type."
            },
```

I could also find useful use cases for objects, for example.

Also, the return value also needs to be changed.